source/core/src/main/com/csse3200/game/areas/MapHandler.java

```diff
@@ -93,32 +85,55 @@ public static void unlockNextArea() {
    * @return true iff the map is unlocked
    */
   public static boolean getUnlockedOcean() {
```

Comment:
Can some of these functions just be combined and take in an enum, e.g. getUnlocked(MapType type)?